yaksh/views.py

```diff
@@ -730,16 +730,17 @@ def monitor(request, questionpaper_id=None):
     return my_render_to_response('yaksh/monitor.html', context,
                                  context_instance=ci)
 
-
-def get_user_data(username, questionpaper_id=None):
+def get_user_data(user_id, questionpaper_id=None, attempt_number = None):
```

Comment:
Instead of get_user_data in the views, can we have it in the Answerpaper object manager?